.slather.yml

```diff
@@ -1,3 +1,8 @@
+coverage_service: sonarqube_xml
+xcodeproj: StreamChat.xcodeproj
+scheme: StreamChat
+source_directory: Sources/
+output_directory: reports
 ignore:
   - "Tests/**/*"
   - "**/*_Tests.swift"
```

Comment:
@bielikb do we still need `**/*_Tests.swift` & `**/*_Mock.swift` here? Also not sure about `**/*_Vendor.swift`, I can't see any files with that prefix in the codebase.

Reply:
@evsaev @alteral indeed `**/*_Tests.swift`, `**/*_Mock.swif` can be removed.
We've got `Reachability_Vendor` in place, so Id keep the `**/*_Vendor.swift` rule in place for now